src/main/kotlin/org/acejump/session/SessionManager.kt

```diff
@@ -23,9 +23,18 @@ object SessionManager {
    * Starts a new [Session], or returns an existing [Session]
    * if the specified [Editor] already has one.
    */
-  fun start(editor: Editor): Session =
-    sessions.getOrPut(editor) { cleanup(); Session(editor) }
+  fun start(editor: Editor): Session = start(editor, listOf(editor))
 
+  /**
+   * Starts a new multi-editor [Session], or returns an existing [Session] if the specified main [Editor] already has one.
+   * The [mainEditor] is used for typing the search query and tag.
+   * The [jumpEditors] are all editors that will be searched and tagged. The list is ordered so that editors earlier in the list will be
+   * prioritized for tagging in case of conflicts.
+   */
+  fun start(mainEditor: Editor, jumpEditors: List<Editor>): Session {
```

Comment:
You could have `jumpEditors` without the `mainEditor`, I'm not sure how useful it is but it would be for example possible for a plugin like easymotion to add a "start search in right splitter" action, where you type in current editor and search in the one to the right.

But it's up to you, it could be changed into vararg - but I don't think varargs are bytecode-compatible, so you'd probably still need a `start(Editor)` method next to it.

Reply:
Not sure I follow. Why do we need it to be bytecode-compatible, is `start` used externally? If so, it should be marked with `@ExternalUsage`.

Reply:
Hmm okay, let's leave it as-is for now.